cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp

```diff
@@ -20,6 +20,10 @@
 --%>
 <jsp:directive.include file="includes/top.jsp" />
 
+<spring:eval expression="@propertyConfigurer.getProperty('consoleContextpath')" var="consoleContextpath"/>
+<spring:eval expression="@propertyConfigurer.getProperty('loginTicket')" var="loginTicket"/>
+<spring:eval expression="@propertyConfigurer.getProperty('flowExecutionKey')" var="flowExecutionKey"/>
+
```

Comment:
Are you sure loginTicket & flowExecutionKey need special treatment here ? to me only the vars from https://github.com/georchestra/georchestra/blob/19.04/cas-server-webapp/src/main/webapp/WEB-INF/cas-servlet.xml#L70 do...